guides/common/modules/con_managing-red-hat-subscriptions.adoc

```diff
@@ -4,43 +4,71 @@
+* Generates a daily snapshot of usage data across all tracked systems.
+* Displays data in a filterable dashboard at the {RHCloud}.
+* Collects historical data for analytics, purchasing decisions, capacity planning, and risk mitigation.
+* Supports the transition to Simple Content Access (SCA), which enforces subscriptions across the organization rather than on individual systems.
+
+.Configuring your system to use the Subscriptions service
+ifndef::satellite[]
 .Prerequisites
-* Ensure you have a Red{nbsp}Hat subscription manifest.
-ifdef::orcharhino[]
-You can create and download your manifest in the https://access.redhat.com/management/subscription_allocations[Red Hat Customer Portal].
-endif::[]
-ifndef::orcharhino[]
-** If your {Project} is connected, use the {RHCloud} to create the manifest.
-ifdef::satellite[]
-For more information, see {RHDocsBaseURL}subscription_central/1-latest/html-single/creating_and_managing_manifests_for_a_connected_satellite_server/index[Creating and managing manifests for a connected {ProjectServer}] in _Subscription Central_.
-endif::[]
-** If your {Project} is disconnected, use the Red Hat Customer Portal to create the manifest.
-ifdef::satellite[]
-For more information, see {RHDocsBaseURL}subscription_central/1-latest/html/getting_started_with_rhel_system_registration/adv-reg-rhel-using-rhsm_#using_manifests_con[Using manifests for a disconnected Satellite Server] in _Subscription Central_.
-endif::[]
+* Enable the `foreman_rh_cloud` plugin:
+[subs="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# foreman-installer --enable-foreman-plugin-rh-cloud
+----
+
+Connected {Project} Users::
+Configure the `foreman_rh_cloud` plugin to share inventory information with the Hybrid Cloud Console.
+Ensure that the *Automatic Inventory Upload* setting is enabled.
+The plugin enables the subscriptions service to track usage information across connected systems.
+
+You can configure the plugin to omit data that is not needed for subscription tracking, such as host names and IP addresses.
 endif::[]
 
+Disconnected users::
+If you cannot or do not want to connect your system directly, you can export a local summary of your product usage and import it into a CMDB, spreadsheet, or another tracking tool.
+
+Disconnected ({Project})::
+Export usage data in one of the following ways:
+
+** Through the `foreman_rh_cloud` plugin, which generates its report locally and allows you to download it from the {ProjectWebUI}.  
+This report is in JSON format and is most suitable for automated scripts or machine ingestion. 
+You can generate this report by using CLI:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# foreman-rake rh_cloud_inventory:report:generate
+----
+
+** Through the *Host - Installed Products* report by navigating to *Monitor* > *Reports* > *Report Templates*. 
+This report can be exported in multiple formats, including CSV, HTML, and YAML, for use in spreadsheet tools or other applications.
```

Comment:
Is there steps to upload it to console.redhat.com for disconnected customers?

Reply:
Will use the link we agreed to use on Slack.